tests/_data/dummy_projects/with-lockfile/package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "",
+  "name": "with-lockfile",
```

Comment:
out of scope. please put this in another (more fast-moving) pullrequest.
thank you in advance

Reply:
Removed.